packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -235,12 +235,15 @@ export async function wrapRunMigration(context: MigrationContext, exec: Migratio
       if (!(error instanceof Error)) {
         error = new Error(error.toString());
       }
+      const errorMessage = context.currentStep
```

Comment:
Suggestuse assembleError() to convert none FxError to InternalError

Reply:
Updated. Thanks.